### PR TITLE
Add new atreus keymap to planck keyboard

### PR DESCRIPTION
**Pull request by matthyx**
Add new atreus keymap to planck keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by matthyx** [comment]
Do I really need to delete this PR and create a new one from a branch?

**Comment by drashna** [comment]
> Do I really need to delete this PR and create a new one from a branch?

No.  However, the warning/message about it is to help you out, longer term, as working from master can cause issue, over time. 

For now, this is just fine.

File: keyboards/planck/keymaps/atreus/config.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#pragma once
```
Comment (drashna) [inline keyboards/planck/keymaps/atreus/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/planck/keymaps/atreus/keymap.c [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+ * |   Q  |   W  |   E  |   R  |   T  |      |      |   Y  |   U  |   I  |   O  |   P  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   A  |   S  |   D  |   F  |   G  |      |      |   H  |   J  |   K  |   L  |   ;  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   Z  |   X  |   C  |   V  |   B  |   `  |   \  |   N  |   M  |   ,  |   .  |   /  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Esc  | Tab  | GUI  | Shift| Bksp | Ctrl | Alt  |Space | Fun  |   -  |   '  |Enter |
+ * `-----------------------------------------------------------------------------------'
+ */
+[0] = KEYMAP(
```
Comment (drashna) [inline keyboards/planck/keymaps/atreus/keymap.c:32]:
```suggestion
[0] = LAYOUT_ortho_4x12(
```

File: keyboards/planck/keymaps/atreus/keymap.c [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+ * |   !  |   @  |  Up  |   $  |   %  |      |      | PgUp |   7  |   8  |   9  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   (  | Left | Down |Right |   )  |      |      | PgDn |   4  |   5  |   6  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   [  |   ]  |   #  |   {  |   }  |   ^  |   &  |   *  |   1  |   2  |   3  |   +  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Upper| Ins  |      |      |      |      |      |      |      |   .  |   0  |   =  |
+ * `-----------------------------------------------------------------------------------'
+ */
+[1] = KEYMAP(
```
Comment (drashna) [inline keyboards/planck/keymaps/atreus/keymap.c:45]:
```suggestion
[1] = LAYOUT_ortho_4x12(
```

File: keyboards/planck/keymaps/atreus/keymap.c [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+ * | Ins  | Home |      | End  | PgUp |      |      |  Up  |  F7  |  F8  |  F9  | F10  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Del  |      |      |      | PgDn |      |      | Down |  F4  |  F5  |  F6  | F11  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Vol+ |      |      |      |      |      |      |  F1  |  F2  |  F3  | F12  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Vol- |      |      |      |      |      |      |      |PrtSc |ScrLk | Play |
+ * `-----------------------------------------------------------------------------------'
+ */
+[2] = KEYMAP(
```
Comment (drashna) [inline keyboards/planck/keymaps/atreus/keymap.c:58]:
```suggestion
[2] = LAYOUT_ortho_4x12(
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by matthyx** [comment]
@drashna thanks, updated

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**